gpbft/participant.go

```diff
@@ -161,10 +161,13 @@ func (p *Participant) beginInstance() error {
 	if err != nil {
 		return fmt.Errorf("failed fetching power table for instance %d: %w", p.nextInstance, err)
 	}
+	// XXX: Need the the power table for the next instance after this one:
+	// GetCommitteeForInstance(p.nextInstance + 1)
+	data := new(InstanceData)
```

Comment:
We have a few options:

1. Actually ask for the next committee.
2. Return the next power table CID as a part of `GetCommitteeForInstance(p.nextInstance)`.
3. Provide some other method.

@Kubuxu preferences?

Reply:
I've gone with (1) for now.

Reply:
`GetCommitteeForInstance` is called in validation flows.

Essentially, there are two things: committees and everything else we agree on, per instance.

Reply:
FWIW I have used the word "committee" inside the participant to mean "the power table plus beacon", i.e. the stuff used to validate a message. I'm happy to change to a different term if we find one, but let's align the terminology across here. I think committee is good enough not to sweat over.

Reply:
The base tipset is not part of that, since it's not known at message validation time (validation can be some number of instances ahead due to the lookback). Dropping mismatched base tipsets is a separate tiny bit of logic, but can't be in the main validation flow.

Reply:
Now that I've actually read all the code here, I think this instance data should actually be fetched at the same time as the chain to be proposed. It doesn't align with the committee used for validation.